modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4022,6 +4102,39 @@ private void _assertObjectEntriesSize2(long size) throws Exception {
 			objectEntries.toString(), size, objectEntries.size());
 	}
 
+	private long _assertObjectEntryProperties(
+			String actualPicklistObjectFieldValue,
+			String expectedPicklistObjectFieldValue,
+			ObjectDefinition objectDefinition)
+		throws Exception {
```

Comment:
What about renaming this method to:
- _addAndAssertObjectEntryWithPicklistObjectField()

What do you think?

I found 1 example of a method named searchAndAssertLength() in a test class :eyes: